### PR TITLE
Fixes #33793 Template written twice

### PR DESCRIPTION
**Pull request by avitova**

### CODE REVIEW

**Comment by tbrisker** [comment]
ok to test

File: app/views/hosts/_provisioning.html.erb [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <h4><%= _('Templates resolved for this operating system') %></h4>
   <% templates.sort{|t,x| t.template_kind <=> x.template_kind}.each do |tmplt| %>
       <div class="form-group">
-        <div class="control-label col-md-8"> <%= _("%s Template") % tmplt.template_kind %>
+        <div class="control-label col-md-8"> <%= _("%s") % tmplt.template_kind %>
```
Comment (ekohl) [inline app/views/hosts/_provisioning.html.erb]: Is the string interpolation still needed or can it simply be
```suggestion
        <div class="control-label col-md-8"> <%= _(tmplt.template_kind) %>
```

Comment (tbrisker) [inline app/views/hosts/_provisioning.html.erb]: When the string is only an interpolation like here, you don't need the interpolation, you could use the variable directly:
```suggestion
        <div class="control-label col-md-8"> <%= _(tmplt.template_kind) %>
```

**Review by tbrisker — CHANGES_REQUESTED** [review]

**Review by tbrisker — DISMISSED** [review]

**Review by ekohl — DISMISSED** [review]

File: app/views/hosts/_provisioning.html.erb [hunk 2]
```diff
@@ -4,7 +4,7 @@
   <h4><%= _('Templates resolved for this operating system') %></h4>
   <% templates.sort{|t,x| t.template_kind <=> x.template_kind}.each do |tmplt| %>
       <div class="form-group">
-        <div class="control-label col-md-8"> <%= _("%s Template") % tmplt.template_kind %>
+        <div class="control-label col-md-8"> <%= _(tmplt.template_kind) %>
```
Comment (ofedoren) [inline app/views/hosts/_provisioning.html.erb]: Do we need to translate it?..

Comment (ekohl) [reply app/views/hosts/_provisioning.html.erb]: Oh good point, previously it was untranslated.

Comment (avitova) [reply app/views/hosts/_provisioning.html.erb]: Oh, true, we should probably not want to translate this.

**Review by ekohl — APPROVED** [review]
Thanks @ofedoren!

**Review by tbrisker — APPROVED** [review]
Thanks @avitova !

**Comment by ezr-ondrej** [comment]
CP 3.0: b8b24edff

**Pull request merged**